ambari-project/pom.xml

```diff
@@ -242,7 +242,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>18.0</version>
+        <version>20.0</version>
```

Comment:
Is it possible to use version 24.1.1 or 25.0?  There is a CVE open for earlier versions - CVE-2018-10237: https://github.com/google/guava/wiki/CVE-2018-10237.

Reply:
Let me see what bringing in that version does. I bumped it to the minimum version required for the reflections library.